Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.xaml

```diff
@@ -0,0 +1,85 @@
+
+    <Style TargetType="controls:MasterDetailsView" >
+        <Setter Property="Background" Value="{ThemeResource ApplicationPageBackgroundThemeBrush}"/>
+        <Setter Property="BorderBrush" Value="{ThemeResource ApplicationForegroundThemeBrush}"/>
+        <Setter Property="BorderThickness" Value="0"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:MasterDetailsView">
+                    <Border Background="{TemplateBinding Background}"
+                            BorderBrush="{TemplateBinding BorderBrush}"
+                            BorderThickness="{TemplateBinding BorderThickness}">
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="SelectionStates">
+                                <VisualState x:Name="NoSelectionWide"/>
+                                <VisualState x:Name="HasSelection">
+                                    <VisualState.Setters>
+                                        <Setter Target="NoSelectionPresenter.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="NoSelectionNarrow">
+                                    <VisualState.Setters>
+                                        <Setter Target="NoSelectionPresenter.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="WidthStates">
+                                <VisualState x:Name="NarrowState">
+                                    <VisualState.StateTriggers>
+                                        <AdaptiveTrigger MinWindowWidth="320" />
+                                    </VisualState.StateTriggers>
+                                    <VisualState.Setters>
+                                        <Setter Target="MasterColumn.Width" Value="*" />
+                                        <Setter Target="DetailsColumn.Width" Value="0" />
+                                        <Setter Target="DetailsPanel.(Grid.Column)" Value="0" />
+                                        <Setter Target="NoSelectionPresenter.Visibility" Value="Collapsed"/>
+                                        <Setter Target="MasterPanel.BorderThickness" Value="0"/>
+                                        <Setter Target="MasterPanel.Width" Value="NaN"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="WideState">
+                                    <VisualState.StateTriggers>
+                                        <AdaptiveTrigger MinWindowWidth="720" />
```

Comment:
OK. Keeping as is: 
![image](https://cloud.githubusercontent.com/assets/1594689/19842684/e7f482ac-9ee0-11e6-9187-18f6bc2587aa.png)